src/content/docs/en/guides/middleware.mdx

```diff
@@ -200,3 +192,7 @@ greeting response
 auth response
 validation response
 ```
+
+### Handling 404 pages
```

Comment:
```suggestion
## Error pages
```
It is not as recognizable as having "404" in the heading, but we don't have enough to say about 500 pages to give it its own section.

`###` to `##` because it isn't a subsection of "Chaining middleware".

Reply:
Agreed!